commands/orgs/index.js

```diff
@@ -32,8 +24,7 @@ module.exports = {
   needsAuth: true,
   flags: [
     {name: 'json', description: 'output in json format'},
-    {name: 'enterprise', hasValue: false, description: 'filter by enterprise orgs'},
-    {name: 'teams', hasValue: false, description: 'filter by teams'}
```

Comment:
might be good to leave this in as a hidden flag just in case so the CLI won't fail if it's passed in. I doubt anyone is using this flag in a script or anything, but it'd be good to be sure.